include/deal.II/lac/block_vector.h

```diff
@@ -517,6 +518,16 @@ namespace internal
   } /* namespace LinearOperator */
 } /* namespace internal */
 
+
+/** mark dealii::BlockVector< Number > as serial vector
```

Comment:
Well, the first line (double star line) is in doxygen a single line comment, which is shown in a preview. That was the idea behind that, but I'll change that.